### PR TITLE
MES-4823 / Initialise state mgmt cpc

### PR DESCRIPTION
**Pull request by matthew2564**
## Description
Initialise state mgmt for CPC tests

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/view-test-result/cat-adi-part2/components/test-details/test-details.cat-adi-part2.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+test-details-card-adi2 {
+  .exclamation-indicator {
+    width: 26px;
+    height: 26px;
+    float: right;
+  }
+}
```
Comment (tomcrawleyy) [inline src/pages/view-test-result/cat-adi-part2/components/test-details/test-details.cat-adi-part2.scss:7]: was including this intentional?

Comment (matthew2564) [reply src/pages/view-test-result/cat-adi-part2/components/test-details/test-details.cat-adi-part2.scss:7]: yes it was

Comment (tomcrawleyy) [reply src/pages/view-test-result/cat-adi-part2/components/test-details/test-details.cat-adi-part2.scss:7]: alright - happy to approve then, just seemed out of place having adi2 related stuff in here

**Review by tomcrawleyy — APPROVED** [review]

**Review by Cullima — APPROVED** [review]
Nice work!

**Pull request merged**